contents/docs/data/autocapture.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Autocapture
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+PostHog can capture all front-end events automatically using autocapture. This means you do not need to manually add tracking to individual buttons or other parts of your product to get started at analyzing user behavior.
+
+Autocapture is available in our [JavaScript snippet](/docs/getting-started/install?tab=snippet#javascript-snippet-badgerecommendedbadge), [JavaScript SDK](/docs/libraries/js), [React SDK](/docs/libraries/react), or [React Native SDK](/docs/libraries/react-native).
+
+### Capturing additional properties in autocapture events
+
+If you add a data attribute onto an element in the format `data-ph-capture-attribute-some-key={someValue}`, then any autocapture event from that element or one of its children will have the property `some-key: 'someValue'` added to it.
+
+This can be useful when you want to add additional information to autocapture events. For example, with a notification bell:
+
+![a notification bell showing 1 unread notification](../../images/docs/data/autocapture/notification-bell.png)
+
+You can template the HTML for it to include the unread count.
+
+```HTML
+<div
+    onClick={toggleNotificationsPopover}
+    data-ph-capture-attribute-unread-notifications-count={unreadCount}
+>
+```
+
+And the autocapture event for clicks on the bell will include the unread count.
+
+## Limitations of autocapture
+
+While autocapture allows you to track the majority of general events on your website right out of the gate, it is important to note that, for security reasons, PostHog is very conservative regarding `input` tags. In order to prevent passwords or other sensitive data from being collected, very little data is collected from inputs with autocapture.
+
+Specifically, PostHog autocapture will grab only the `name`, `id`, and `class` attributes from `input` tags.
+
+If you need to collect more data from inputs, you should use [custom events and Actions](/docs/user-guides/actions).
+
+## Configuring autocapture
+
+> This section covers configuring autocapture in the JavaScript and React SDKs. For the React Native SDK, see our [React Native docs](/docs/libraries/react-native#autocapture).
+
+### Reducing events
+
+Autocapture enables you to start capturing events on your site quickly, but this can lead to large numbers of events. To counteract this, you can configure autocapture using allowlists. Allowlists are an array of allowed events – any events for elements not matching an item in the array will be dropped.
```

Comment:
```suggestion
Autocapture enables you to start capturing events on your site quickly, but this can lead to large numbers of events. To counteract this, you can configure autocapture using allowlists. Allowlists are an array of allowed events – any events for elements not matching an item in the array are dropped.
```